.github/workflows/build.yml

```diff
@@ -11,7 +11,14 @@ jobs:
         uses: actions/setup-java@v1
         with:
           java-version: 1.8
-      - name: Grant execute permission for gradlew
-        run: chmod +x gradlew
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-cache-v1-${{ hashFiles('**/*.gradle*', '**.gradle-wrapper.properties') }}
```

Comment:
why v1? we are using cache@v2 right?

Reply:
actually this is just the cache key name that we will sace and use. before, I'm using `gradle-cache`, but its failed to use that cache and we can not clear that. do we need to use v2 for better cache naming?

Reply:
we can use any static word here, but it's just that v1 can be misleading

Reply:
changing this to `dagger-cache`